lib/utils/defer.js

```diff
@@ -0,0 +1,16 @@
+/**
+ * Promisify a function of the form (value, options, cb)
+ * @param  {Function} asyncFunc The function
+ * @return {Function} 					The new function, as a Promise
```

Comment:
It's the tabbing. JSDocs pushes over the description so that it lines up with the previous line's description